huxley/core/admin/delegate.py

```diff
@@ -29,6 +30,29 @@ def roster(self, request):
 
         return roster
 
+    def load(self, request):
+        '''Loads new Assignments.'''
+        delegates = request.FILES
+        reader = csv.reader(delegates['csv'])
+
+        assignments = {}
+        for assignment in Assignment.objects.all():
+            assignments[assignment.committee.name.encode('ascii', 'ignore'),
```

Comment:
This code style is kinda weird and inconsistent.
```python
assignments[
    foo,
    bar,
    baz,
] = assignment
```

This way the indentation is consistent with the meaning of each expression.